hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateSpecs.java

```diff
@@ -137,8 +139,26 @@ public List<HapiSpec> getSpecsInSuite() {
                 canUpdateExpiryOnlyOpWithoutAdminKey());
     }
 
+    private List<HapiSpec> negativeTests() {
+        return List.of(
+                updateTokenTreasuryRequiresZeroTokenBalance(),
```

Comment:
Need more tests covering updating multiple keys in same transaction

Reply:
@Neeharika-Sompalli 
Those types of tests are in `Hip540ChangeOrRemoveKeysSuite` file. We split the specific tests about the HIP there for better visibility